Code/GraphMol/MolStandardize/MolStandardize.cpp

```diff
@@ -64,7 +64,6 @@ RWMol *fragmentParent(const RWMol &mol, const CleanupParameters &params,
   LargestFragmentChooser lfragchooser(params.preferOrganic);
   ROMol nm(*cleaned);
   ROMOL_SPTR lfrag(lfragchooser.choose(nm));
-  delete cleaned;
```

Comment:
Removing this line is clearly correct when skip_standardize is true, but introduces a core leak when `skip_standardize` is false and we've called `cleanup()`.
A better solution here would be:
```
if(!skip_standardize){
  delete cleaned;
}
```